test/e2e/hotGraphic.cy.js

```diff
@@ -0,0 +1,54 @@
+describe('Hot Graphic', function () {
+  function loopThroughHotGraphic(hotGraphicComponent) {
+    const itemsCount = Object.keys(hotGraphicComponent._items).length
+    cy.get('.hotgraphic__pin-item').should('have.length', itemsCount)
+    for (let i = 0; i < itemsCount; i++) {
+      cy.get(`.hotgraphic__pin-item .item-${i}.is-visited`).should('not.exist')
+      cy.get('.notify__popup.hotgraphic').should('not.exist')
+      cy.get(`.hotgraphic__pin-item .item-${i}`).click()
+      cy.get('.notify__popup.hotgraphic').should('be.visible')
+      cy.testContainsOrNotExists('.hotgraphic-popup__item-title-inner', hotGraphicComponent._items[i].title)
+      cy.testContainsOrNotExists('.hotgraphic-popup__item-body-inner', hotGraphicComponent._items[i].body)
+      cy.get('button.hotgraphic-popup__close').click()
+      cy.get(`.hotgraphic__pin-item .item-${i}.is-visited`).should('exist')
+    }
```

Comment:
might want to put this into a test called something like 'Can view all hotgraphic pin items'